calcom/cal.com
# fix: Team list now aware of org teams

## What does this PR do?

Teams list now shows correct URL for org sub-teams and created sub-teams now creates membership for owner so they can see the teams in the page.

Fixes #9542 


## Type of change

- New feature (non-breaking change which adds functionality)

## How should this be tested?

Create an org with sub-teams from wizard and go to Teams page, see the correct URL and that org team per se is not shown.

## Mandatory Tasks
